onnx/shape_inference/implementation.h

```diff
@@ -17,23 +17,32 @@ struct GraphInferenceContext {
       const std::unordered_map<std::string, TypeProto*>&
           outer_scope_value_types_by_name_in,
       const std::unordered_map<std::string, int> opset_imports_in,
+      SymbolTable& symbolTable_in,
       const ISchemaRegistry* schema_registry_in = OpSchemaRegistry::Instance())
       : outer_scope_value_types_by_name{&outer_scope_value_types_by_name_in},
         opset_imports{opset_imports_in},
-        schema_registry{schema_registry_in} {}
+        schema_registry{schema_registry_in},
+        symbolTable{symbolTable_in} {}
 
 
   const std::unordered_map<std::string, TypeProto*>*
       outer_scope_value_types_by_name;
   const std::unordered_map<std::string, int> opset_imports;
   const ISchemaRegistry* schema_registry;
+  SymbolTable& symbolTable;
 
+  SymbolTable* mutableSymbolTable() {
```

Comment:
I dont understand the purpose of this... what is this for?

Reply:
Actually we can simply use symbolTable above... Just removed it. Thanks!